tests/test_flow.py

```diff
@@ -1938,7 +1940,7 @@ def test_error_in_concurrent_by_key_task():
         WriteToTable(table, columns=['twice_total_activities']),
     ]).run()
 
-    controller.emit(None)
+    controller.emit({'col1': 0}, 'tal')
```

Comment:
Aw

Reply:
you can't emit an event without a key (it throws an exception now)

Reply:
Tal 🥰